chart/kubeapps/values.yaml

```diff
@@ -124,6 +124,18 @@ chartsvc:
     # requests:
     #  cpu: 100m
     #  memory: 128Mi
+  livenessProbe:
+    httpGet:
+      path: /live
```

Comment:
Is this route (/live) based on any common practice? I've seen the proliferation of healthz instead

Reply:
I see, you are just just using them.

Reply:
@migmartri when I implemented this in chartsvc, I used the https://github.com/heptiolabs/healthcheck library that suggests using `/live` and `/ready` https://github.com/heptiolabs/healthcheck#usage. Specifically, the difference from using `/healthz` is having two separate endpoints that might have different implementations of the healthcheck (e.g. `/ready` might check that MongoDB is connectable, though not currently implemented).

Reply:
Oh nice, I was not aware of that library, thanks for the info.

> Specifically, the difference from using a single /healthz is having two separate endpoints that might have different implementations of the healthcheck (e.g. /ready might check that MongoDB is connectable, though not currently implemented).

Right, in the past I've been handling this by passing a `liveness` parameter to the /healthz endpoint.

Reply:
> @migmartri makes sense, yeah, the name of the endpoints doesn't matter much here I guess, although I agree that `healthz` is more common. `/live` and `/ready` are also possibly too specific to Kubernetes, although the concept does transfer across. Either way, probably not important enough to change.

No it's not important, thanks for the info.